src/fidesops/ops/schemas/saas/strategy_configuration.py

```diff
@@ -87,6 +91,33 @@ class CursorPaginationConfiguration(StrategyConfiguration):
     field: str
 
 
+class ApiKeyAuthenticationParameter(QueryParam):
+    method: Optional[List[HTTPMethod]]
```

Comment:
This took me a while but I see that we use the method to indicate if the headers/query_params should be added to a request. Did we run into this use case with the connectors already?

Reply:
we did run into this with [Rollbar](https://github.com/ethyca/fidesops/pull/1227), but per discussion with @galvana , we decided to de-scope this feature from the PR/issue, and allow Rollbar to continue to be configured "manually" via query param. Adding this functionality to the auth strategy seemed beyond the scope of what auth strategies are meant to do.